conf/test_route_control.py

```diff
@@ -1,3 +1,9 @@
+#!/usr/bin/env python
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2023 Canonical Ltd.
+# Copyright 2023 Open Networking Foundation
```

Comment:
This line does not have to be here because this file is new and created by your, so, I think only the Canonical copyright should be in the new files

Reply:
Awesome, I fixed it thank you.